.github/workflows/pr.yml

```diff
@@ -0,0 +1,72 @@
+name: Pull Request
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions: {}
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  # TODO: Remove when pnpm is exclusively used.
+  ASPECT_RULES_JS_FROZEN_PNPM_LOCK: '1'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Initialize environment
+        uses: angular/dev-infra/github-actions/npm/checkout-and-setup-node@db91da4e742cd081bfba01db2edc4e816018419b
+      - name: Install node modules
+        run: yarn install
+      - name: Execute Linting
+        run: yarn bazel test --test_tag_filters=+lint //...
```

Comment:
Is there `+` supposed to be there on the filter?

I havent seen that syntax before.

Reply:
Was there in the original command line but doesn't seem to be mentioned in the bazel docs.

Removed.